lib/api_proxy.rb

```diff
@@ -0,0 +1,13 @@
+require 'rack-proxy'
+
+class ApiProxy < Rack::Proxy
+  def rewrite_env(env)
+    env["HTTP_HOST"] = "api.et.127.0.0.1.nip.io"
```

Comment:
We still have the full system url hard coded in here.  This should be calculated from the API_URL env var I think it is ?  same with the line below

Reply:
Thanks, good idea. It's `ET_API_URL` and has been fixed in 24c3403